### PR TITLE
chore: add dsn for connection to mysql

### PR DESCRIPTION
**Pull request by kmlebedev**
# What problem are we solving?

it is necessary to pass arbitrary connection param to the msql driver like [tls](https://github.com/go-sql-driver/mysql#tls)

# How are we solving the problem?

add dsn to filer config

# How is the PR tested?

localy

# Checks
- [ ] I have added unit tests if possible.
- [ ] I will add related wiki document changes and link to this PR after merging.

### CODE REVIEW

File: weed/command/scaffold/filer.toml [hunk 1]
```diff
@@ -60,6 +60,8 @@ connection_max_idle = 2
 connection_max_open = 100
 connection_max_lifetime_seconds = 0
 interpolateParams = false
+#     [username[:password]@][protocol[(address)]]/dbname[?param1=value1&...&paramN=valueN]
+dsn = "root@tcp(localhost:3306)/seaweedfs?collation=utf8mb4_bin"
```
Comment (chrislusf) [inline weed/command/scaffold/filer.toml]: move the new section to after line 53, and add comments that this dsn will take priority over "hostname, port, username, password, database".

**Comment by kmlebedev** [comment]
@chrislusf done

**Comment by chrislusf** [comment]
To adapt with password, the password needs to be set first? If so, need to mention it in the comment.

**Comment by kmlebedev** [comment]
> password needs to be

make it so you don't need to set a password

**Pull request merged**